### PR TITLE
Remove jQuery from app

### PR DESCRIPTION
**Pull request by rwwagner90**
Also had to switch to a new context menu plugin. This should get us closer to #590

screen shot 2018-03-08 at 10 45 34 am

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ module.exports = {
   env: {
     browser: true
   },
+  globals: {
+    basicContext: false
```
Comment (rwjblue) [inline .eslintrc.js]: 🤔 what is `basicContext`?

File: app/components/drag-handle.js [hunk 2]
```diff
@@ -51,32 +50,18 @@ export default Component.extend({
...
-      position -= this.get('left');
-      if (position >= this.get('minWidth') && position <= this.get('maxWidth')) {
-        this.set('position', position);
-        this.get('on-drag')(position);
-      }
-    })
-      .on(`mouseup.${namespace} mouseleave.${namespace}`, () => {
-        this.stopDragging();
-      });
+    document.body.addEventListener(`mousemove.${namespace}`, this._mouseMoveHandler);
```
Comment (rwjblue) [inline app/components/drag-handle.js]: These also need to be cleaned up in`willDestroy` or `willDestroyElement`

File: package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "aws-sdk": "^2.3.3",
     "babel-eslint": "^8.2.2",
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
+    "basiccontext": "^3.5.1"
```
Comment (rwjblue) [inline package.json]: missing a comma here

**Comment by teddyzeenny** [comment]
Thanks! Can you please add a screenshot of the new context menu?

File: .eslintrc.js (hunk 1 above)
Comment (rwwagner90) [reply .eslintrc.js]: It is the new contextMenu plugin we are using.

File: app/components/drag-handle.js (hunk 2 above)
Comment (rwwagner90) [reply app/components/drag-handle.js]: `willDestroyElement` calls `this.stopDragging();` which has the `removeEventListener` calls. Did I mess them up?

File: package.json (hunk 3 above)
Comment (rwwagner90) [reply package.json]: 👍

**Comment by rwwagner90** [comment]
@teddyzeenny I added a screenshot 👍

File: app/components/drag-handle.js (hunk 2 above)
Comment (rwjblue) [reply app/components/drag-handle.js]: Ahh, I see, thank you for explaining!

**Pull request merged**